avocado/core/test.py

```diff
@@ -1093,13 +1094,40 @@ def fetch_asset(self, name, asset_hash=None, algorithm=None,
         :param locations: list of URLs from where the asset can be
                           fetched (optional)
         :param expire: time for the asset to expire
-        :raise EnvironmentError: When it fails to fetch the asset
```

Comment:
This is a nipick, but the change from EnvironmentError to OSError is really a documentation fix because of 5690df858a8611d79fcca709d3cc49b3fa9b2687, right? If so, keeping it out of this commit (in a separate one) is the best course of action.

Reply:
Okay, added it to its own commit on v3.